.travis.yml

```diff
@@ -1,5 +1,6 @@
 language: elixir
 elixir:
+  - 1.3.1
```

Comment:
Ah, that makes sense. You're right though, no reason it can't support 1.3.x and above.